src/operator/quantization/mkldnn/mkldnn_quantized_fully_connected.cc

```diff
@@ -52,15 +47,15 @@ void MKLDNNQuantizedFullyConnectedForward(const nnvm::NodeAttrs &attrs,
   NDArray weight = in_data[fullc::kWeight];
 
   const float min_data =
-    in_data[num_inputs + quantized_fc_enum::kDataMin].data().dptr<float>()[0];
+    in_data[num_inputs + quantized_fullc::kDataMin].data().dptr<float>()[0];
```

Comment:
Quite strange usage. Why not define a whole input sets with original inputs?

Reply:
Original inputs might not include bias, which results in different index for all these min/max. Just to simplify the ordering for quantized op only.